tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -206,8 +221,82 @@ object CompletionArgSuite extends BaseCompletionSuite {
         |""".stripMargin,
     """|argument: Int
        |argument = : Int
+       |argument = argument : Int
+       |""".stripMargin,
+    topLines = Some(3)
+  )
+
+  check(
+    "named-multiple",
+    s"""|object Main {
+        |  def foo(argument : Int) : Int = argument
+        |  val number = 1
+        |  val number2 = 2
+        |  val number4 = 4
+        |  val number8 = 8
+        |  foo(ar@@)
+        |}
+        |""".stripMargin,
+    """|argument = : Int
+       |argument = number : Int
```

Comment:
I am concerned we're trying to be too smart in this particular case. I think it's fine to complete only one thing at a time, for example there may be variables in scope that start with `arg*`.

Reply:
We first check for values with the name, then for named arguments, named arguments with values and the autocomplete is last.

I think that's the best option and is compatible with the current behaviour. Wouldn't cause users to need to scroll more then now to use the current capabilities.